data-model/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM nginx:stable-alpine3.17-slim
+
+COPY ./nginx/nginx.conf /etc/nginx/nginx.conf
+
+COPY ./annotations/0.1.0/schema/annotation.json schema-root/schemas/annotations/0.1.0/annotation.json
+
+COPY ./fdo-profiles/0.1.0/digital-specimen/schema/ schema-root/schemas/fdo-profiles/0.1.0/
+COPY ./fdo-profiles/0.1.0/doi-kernel/schema/ schema-root/schemas/fdo-profiles/0.1.0/
+COPY ./fdo-profiles/0.1.0/handle-kernel/schema/ schema-root/schemas/fdo-profiles/0.1.0/
+
+COPY ./specimens-and-media/0.1.0/digital-specimens/schema schema-root/schemas/digital-specimens/0.1.0/
+COPY ./specimens-and-media/0.1.0/shared-models/schema schema-root/schemas/shared-models/0.1.0/
+COPY ./specimens-and-media/0.1.0/digital-media-object/schema schema-root/schemas/digital-media-object/0.1.0/
```

Comment:
I feel like this might be a lot of work to go over all the versions when the schemas get updated.
Is there a way you could automatically include these files by the latest version and dynamically make the routes?

Reply:
Yeah exactly. The trouble with serving static content :upside_down_face: